### PR TITLE
Chapter 5: Filling in the layout

### PR DESCRIPTION
**Pull request by hungvuongdora**

### CODE REVIEW

**Comment by hungvuongdora** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,101 @@
+@import "bootstrap-sprockets";
```
Comment (DuongNT1512) [inline app/assets/stylesheets/custom.scss]:
```suggestion
@import 'bootstrap-sprockets';
```

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About",   '#' %></li>
```
Comment (DuongNT1512) [inline app/views/layouts/_footer.html.erb]:
```suggestion
      <li><%= link_to "About", "#" %></li>
```

**Comment by hungvuongdora** [comment]
ready

**Comment by hungvuongdora** [comment]
ready

**Comment by hungvuongdora** [comment]
ready

**Comment by hungvuongdora** [comment]
ready

**Review by DuongNT1512 — COMMENTED** [review]
merged

**Comment by DuongNT1512** [comment]
merged

**Pull request merged**